src/main/xcontainers/XPanels.js

```diff
@@ -63,7 +64,7 @@ class XPanels extends Component {
   }
 };
 
-const mapStateToProps = (state) => {
+const mapStateToProps = (state, ownProps) => {
```

Comment:
Why add this parameter if it is never used?

Reply:
We thought we would need it, then we realized we wont and forgot to remove it